rest/src/main/kotlin/builder/message/MessageCreateBuilder.kt

```diff
@@ -40,6 +40,9 @@ class MessageCreateBuilder : RequestBuilder<MultipartMessageCreateRequest> {
     private var _allowedMentions: Optional<AllowedMentionsBuilder> = Optional.Missing()
     var allowedMentions: AllowedMentionsBuilder? by ::_allowedMentions.delegate()
 
+    private var _components: Optional<MutableList<DiscordComponent>> = Optional.Missing()
+    var components: MutableList<DiscordComponent>? by ::_components.delegate()
```

Comment:
Make this a val, empty list implies unset

Reply:
this breaks the components function

Reply:
It doesn't, it's a mutable list, you can just add to it